content/pt-br/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -0,0 +1,285 @@
+
+## {{% heading "prerequisites" %}}
+
+Esse tutorial assume que você esta usando `minikube` para rodar um cluster Kubernetes local.
+Visite [Install tools](/docs/tasks/tools/#minikube) para aprender como instalar o `minikube`.
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+Se você estiver usando uma versão mais antiga do Kubernetes, veja a documentação para essa versão.
+
+### Criando um cluster minikube
+Se você ainda não configurou um cluster local, rode `minikube start` para criar um cluster.
+
+<!-- steps -->
+
+## Ativando o Ingress controller
+
+1. Para ativar o NGINX Ingress controller, rode os seguintes comandos:
+
+   ```shell
+   minikube addons enable ingress
+   ```
+
+1. Verifique que o NGINX Ingress controller esta rodando
+
+   ```shell
+   kubectl get pods -n ingress-nginx
+   ```
+
+   {{< note >}}
+   Pode demorar alguns minutos antes dos pods estarem rodando OK.
+   {{< /note >}}
+
+   O resultado deve ser similar a:
+
+   ```none
+   NAME                                        READY   STATUS      RESTARTS    AGE
+   ingress-nginx-admission-create-g9g49        0/1     Completed   0          11m
+   ingress-nginx-admission-patch-rqp78         0/1     Completed   1          11m
+   ingress-nginx-controller-59b45fb494-26npt   1/1     Running     0          11m
+   ```
+
+## Deployando uma aplicação hello world
+
+1. Crie um Deployment usando o seguinte comando:
+
+   ```shell
+   kubectl create deployment web --image=gcr.io/google-samples/hello-app:1.0
+   ```
+
+   O resultado deve ser:
+
+   ```none
+   deployment.apps/web created
+   ```
+
+1. Exponha o Deployment:
+
+   ```shell
+   kubectl expose deployment web --type=NodePort --port=8080
+   ```
+
+   O resultado deve ser:
+
+   ```none
+   service/web exposed
+   ```
+
+1. Verifique que o Service esta criado e disponível em uma porta do node:
+
+   ```shell
+   kubectl get service web
+   ```
+
+   O resultado deve ser similar:
+
+   ```none
+   NAME      TYPE       CLUSTER-IP       EXTERNAL-IP   PORT(S)          AGE
+   web       NodePort   10.104.133.249   <none>        8080:31637/TCP   12m
+   ```
+
+1. Visite o Service via NodePort:
+
+   ```shell
+   minikube service web --url
+   ```
+
+   O resultado é similar a:
+
+   ```none
+   http://172.17.0.15:31637
+   ```
+
+   ```shell
+   curl http://172.17.0.15:31637 
+   ```
+
+   O resultado é similar a:
+
+   ```none
+   Hello, world!
+   Version: 1.0.0
+   Hostname: web-55b8c6998d-8k564
+   ```
+
+   Você agora pode acessar a aplicação de exemplo através do endereço IP do Minikube e NodePort.
+   O proximo passo te permite acessar a aplicação usando o recurso Ingress.
+
+## Criando um Ingress
+O manifesto a seguir define um Ingress que envia tráfego para seu Serviço via
+`hello-world.info`.
+
+1. crie `example-ingress.yaml` usando o arquivo:
+
+   {{% code_sample file="service/networking/example-ingress.yaml" %}}
```

Comment:
It seems the PR build preview is encountering [failures](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65f88ef46e8c3e000888ff2f) as the `{{ code_sample }}` shortcode is unable to locate the "example-ingress.yaml" for Portuguese localization.

Build Errors [(see here)](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65f88ef46e8c3e000888ff2f):

> 7:00:03 PM: Start building sites …
7:00:03 PM: hugo v0.121.2-6d5b44305eaa9d0a157946492a6f319da38de154+extended linux/amd64 BuildDate=2024-01-05T12:21:15Z VendorInfo=gohugoio
7:01:31 PM: ERROR [pt-br] "example-ingress.yaml" not found in "docs/tasks/access-application-cluster/service/networking"
7:01:39 PM: Failed during stage "building site": Build script returned non-zero exit code: 2


To address this, I suggest copying the `example-ingress.yaml` file under [content/en/examples/service/networking/example-ingress.yaml](https://github.com/kubernetes/website/blob/main/content/en/examples/service/networking/example-ingress.yaml) to a similar location for Portuguese by creating the "example-ingress.yaml" file under "[content/pt-br/examples/service/networking](https://github.com/kubernetes/website/tree/main/content/pt-br/examples/service/networking)". This would likely resolve this error.

Reply:
Hi there, @matheusde! Could you please create the `example-ingress.yaml` file in the correct path as described by @dipesh-rawat in the previous comment, so the build gets fixed?